tests/test_data/test_data.py

```diff
@@ -0,0 +1,35 @@
+import json
+import os
+
+
+class MockResponse:
+    def __init__(self, json_data, status_code):
+        self.json_data = json.loads(json_data)
+        self.status_code = status_code
+        self.ok = status_code < 300
+
+    def json(self):
+        return self.json_data
+
+
+dirname = os.path.dirname(__file__)
+
+with open(f'{dirname}/peak_faction_holiday_response.json', mode='r') as data:
+    holiday_response = data.read()
+
+with open(f'{dirname}/peak_faction_workingday_response.json', mode='r') as data:
+    workingday_response = data.read()
+
+
+def mocked_requests_holiday_response(*args, **kwargs):
+    if args[0] == 'https://elitebgs.app/api/ebgs/v5/factions' \
+            and kwargs['params']['name'] == 'Rackham Capital Investments':
+        return MockResponse(holiday_response, 200)
+    return MockResponse(None, 404)
```

Comment:
The second case here seems unused in the testing

Reply:
2 cases, 2 tests - both are used

Reply:
You have 2 tests and 2 functions providing 4 states.  Either you are missing tests for 2 of the states, or they should not exist.

Reply:
Referring to `return MockResponse(None, 404)` as a "second case" is confusing. 

Please share a message like "consider removing this line, as the control should never reach it" so it's clear.

I removed it, since the test will work expected whether it's 404 or Error.

And I see a broad catch-all right here : https://github.com/PilotsTradeNetwork/BoozeBot/blob/dev/ptn/boozebot/PHcheck.py#L15

What's worse is that it doesn't even log the actual exception...